gdnative/src/gdscript.rs

```diff
@@ -0,0 +1,30 @@
+use gdnative_bindings::{Resource, ResourceLoader};
+use gdnative_core::{
+    core_types::NodePath,
+    object::{memory::RefCounted, GodotObject, Ref, SubClass},
+};
+
+/// Loads a resource from the filesystem located at `path`. The resource is loaded on the method call (unless it's referenced already elsewhere, e.g. in another script or in the scene), which might cause slight delay, especially when loading scenes. To avoid unnecessary delays when loading something multiple times, either store the resource in a variable or use `preload()`.
+/// # Note:
+/// Resource paths can be obtained by right-clicking on a resource in the FileSystem dock and choosing "Copy Path" or by dragging the file from the FileSystem dock into the script.
+/// # Load a scene called main located in the root of the project directory and cache it in a variable. var main = load("res://main.tscn")  main will contain a PackedScene resource.
+/// # Important:
+/// The path must be absolute, a local path will just return null.
+/// This method is a simplified version of `ResourceLoader.load()`, which can be used for more advanced scenarios.
+/// # Examples:
+/// ```no_run
+/// use gdnative::globalscope::load;
+/// use gdnative::prelude::PackedScene;
+///
+/// let scene = load::<PackedScene>("res://path").unwrap();
+/// ```
```

Comment:
RustDoc uses Markdown-like syntax, so each `#` is interpreted as a header.

Your doc renders as follows, I don't think that's intended 🙂 
![grafik](https://user-images.githubusercontent.com/708488/188671349-27412304-997b-4da5-bde3-6919161ba995.png)